### DISCUSSION TITLE
Bit Rate Change

### DISCUSSION THREAD

**Discussion by ghost** [subdavis/kobo-book-downloader#56] [Q&A] [answered]
Hello!

It's been some months since I've used the tool and I've noticed the default bit rate for audiobook downloads has changed from 192kbps to 64kbps. Thinking it might just be an isolated incident, I redownloaded a previous audiobook and sure enough these files are 64kbps while the original are 192kbps. 

Has anyone else noticed this and is there a way to get higher bit rate again?

**Comment by subdavis** [comment]
Interesting, thanks for the report!  Not sure what would be causing it.

**Reply by ghost** [reply]
Unfortunately it looks like Kobo's restricting all downloads to 64kbps across the board. In my iOS app I had a file saved to iPad that was 2688.79 MB. Removed it and downloaded again and now it's only 896.27 MB. Doesn't seem to be anyway to change it in app settings.

**Comment by bybpow** [comment, marked as answer]
I can also verify this is true. All files are now encoded in 64kbps.
